conjure_oxide/tests/integration/minion_constraints/div_undefzero-02-zero/input.eprime

```diff
@@ -0,0 +1,7 @@
+language ESSENCE' 1.0
+
+find x : int(1..100)
+find z : int(1..2)
+
+$ x/0 is always true 
```

Comment:
technically x/0 is always undefined. and `x/0=z` is always false.

Reply:
a/b=c
~~>
div_undefzero(a,b,c) /\ b != 0


and if this is not at the top level, you want bubbles. so more like:

a/b=c
~~> undefinedness
{ SafeDiv(a,b) @ b != 0 } = c
~~> bubble up
{ SafeDiv(a,b) = c @ b != 0}
~~> bubble to and
SafeDiv(a,b) = c /\ b != 0
~~> remove safediv
{ div_undefzero(a,b,c) /\ b != 0}

where {x @ B} is syntax for x, with bubble B (a boolean expression) attached.

Reply:
Ah, i get it now!

We are adding our own undefinedness semantics on top of div.

I'll make this clear in the test.